smart_campus/app/templates/app/reward_list_page.html

```diff
@@ -0,0 +1,46 @@
+{% extends "app/base.html" %}
+{% load static %}
```

Comment:
You add one blank line between `extend` and `load static` in some files.
You should unify the style.
I  suggest that you don't add one in this case.

Reply:
Okay, I will remove other files with this line.